src/App.jsx

```diff
@@ -1,40 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { getSelectedTab } from './components/helpers/helpers';
```

Comment:
The method getSelectedTab is not defined in the helpers file. Please add the function definition in the helpers file.